conduct.md

```diff
@@ -68,33 +68,35 @@ TODO: Write an introductory paragraph.
 
 ### Decisions
 
-TODO: Write an introductory paragraph.
+Decisions are the action or process of deciding something or of resolving a question. Decisions are the process of making choices by identifying a decision, gathering information, and assessing alternative resolutions.
 
 #### Acceptable Conduct
 
-- Create decision that mutually benefit each other (Schedules, Coding). This is to help understand each other's responsibilities and respect on what is possible for both individual interest and the team interest. 
+- Create decision that mutually benefit each other (Schedules, Coding). This is to help understand each other's responsibilities and respect on what is possible for both individual interest and the team interest.
 - TODO
 - TODO
 
 #### Unacceptable Conduct
 
-- Never informing on others on unattended appointments. This will only create more problems as other team members has to do more work unexpectedly, therefore the work will not be as good as intended to be. 
+- Never informing on others on unattended appointments. This will only create more problems as other team members has to do more work unexpectedly, therefore the work will not be as good as intended to be.
 - TODO
 - TODO
 
 ### Postmortems
 
-TODO: Write an introductory paragraph.
+Postmortems is an analysis or discussion of an event held soon after it has occurred, especially in order to determine
+why it was a failure. A post-mortem is where a team reflects on what went wrong with something they did, and documents
+it and/or amends their process to stop it happening again.
 
 #### Acceptable Conduct
 
-- Always be transparent on what are we doing wrong with. The reason being is because to create a mutual relationship environment between the team to understand the good and the bad of the activity together. 
-- TODO
+- Always be transparent on what are we doing wrong with. The reason being is because to create a mutual relationship environment between the team to understand the good and the bad of the activity together.
+- TODOz
 - TODO
 
 #### Unacceptable Conduct
 
-- Never create unnecessary mistakes (Forget to git pull, never creating pull requests). This is very important to not create more problems and not to halt on the team's progress on the work. 
+- Never create unnecessary mistakes (Forget to git pull, never creating pull requests). This is very important to not create more problems and not to halt on the team's progress on the work.
```

Comment:
This does not seem like a conduct specific to postmortems. Conduct in "Postmortems" section typically refers to what not to do during postmortems. Here is a reference that might be helpful in understanding postmortems: https://www.atlassian.com/incident-management/handbook/postmortems